Automattic/wp-calypso
# Refactored the Collection components to delete un-needed components. …

This PR cleans up some unnecessary code and configuration keys after the latest stakeholder walkthrough

## Proposed Changes

* Deleted the ThemeCollectionsLayout component.
* Removed the randomization inside collections.
* Removed randomization config keys.

## Testing Instructions



* Open the Calypso Live link
* Go to the LiTS and LoTS and add the corresponding flags
* `flags=themes/discovery-lots` or `flags=themes/discovery-lits`
* Everything should work as expected without errors.

No screenshots provided since there are no visual changes.

## Pre-merge Checklist
